internal/operands/metrics/resources.go

```diff
@@ -76,6 +76,22 @@ func newPrometheusRule(namespace string) *promv1.PrometheusRule {
 								"severity": "Warn",
 							},
 						},
+						{
+							Record: "kubevirt_ssp_total_restored_common_templates",
+							Expr:   intstr.FromString("sum(increase(total_restored_common_templates{pod=~'ssp-operator.*'}[1h]))"),
+						},
+						{
+							Alert: "SSPCommonTemplatesModificationReverted",
+							Expr:  intstr.FromString("kubevirt_ssp_total_restored_common_templates > 0"),
```

Comment:
@borod108 Is there a way that "kubevirt_ssp_total_restored_common_templates" will return no values?

Reply:
I do not think so. If it is registered it will return a value.

Reply:
@borod108 Where did you add the OR vector(0) ?